docs/reference/release-notes/8.0.0.asciidoc

```diff
@@ -0,0 +1,699 @@
+[[release-notes-8.0.0]]
+== {es} version 8.0.0
+
+coming::[8.0.0]
+
+The following list are changes in 8.0.0 as compared to 7.17.0, and combines
+release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+Also see <<breaking-changes-8.0,Breaking changes in 8.0>>.
+
+[[breaking-8.0.0]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* Percentiles aggregation: disallow specifying same percentile values twice {es-pull}52257[#52257] (issue: {es-issue}51871[#51871])
+* Remove Adjacency_matrix setting {es-pull}46327[#46327] (issues: {es-issue}46257[#46257], {es-issue}46324[#46324])
```

Comment:
Nit: better as this?
```suggestion
* Remove adjacency matrix setting {es-pull}46327[#46327] (issues: {es-issue}46257[#46257], {es-issue}46324[#46324])
```

Reply:
Thanks! Fixed in https://github.com/elastic/elasticsearch/pull/83689/commits/b4fbb7ace5e3ebf3fe1a3b9f1685d7d5815a160c